google-map-marker.html

```diff
@@ -299,6 +332,7 @@
       this._clickEventsChanged();
       this._contentChanged();
       this._mouseEventsChanged();
+      this._openChanged();
```

Comment:
I tried removing this but the problem is that when _openChanged is called because the open attribute is set the infowindow object hasn't been created yet.